packages/modules/imodel-browser/src/utils/_buildMenuOptions.tsx

```diff
@@ -29,15 +29,17 @@ export const _buildManagedContextMenuOptions: <T>(
     })
     .map(({ key, visible, onClick, ...contextMenuProps }) => {
       return (
-        <MenuItem
-          {...contextMenuProps}
-          onClick={() => {
-            closeMenu?.();
-            onClick?.(value, refetchData);
-          }}
-          key={key}
-          value={value}
-        />
+        // MenuItem doesn't expose the real click event, so wrap it in div to intercept it and stop propagation
```

Comment:
cc @mayank99

Reply:
I made a sandbox showing the issue:
https://codesandbox.io/p/devbox/react-typescript-forked-lx3tjr?workspaceId=ws_MVem6KqRQa8R4spaHpry4y

I think the click shouldn't be propagated by default, but it would be nice to expose the click event so we have some control over it.

Reply:
This is an issue we are aware of. The recommended solution is to stop the propagation on the `<DropdownMenu>` rather than adding `<div>`s around the `<MenuItem>`s.

![Screenshot of code](https://github.com/user-attachments/assets/0dfa0985-4285-4bf2-8ba4-6ff118e93fb2)

Reply:
Moved it.

If you're not going to fix this soon, it should really be mentioned in your documentation and website examples. This took a while to diagnose...

Reply:
We are working on proper documentation for Table. In the meantime, the [Table documentation](https://itwinui.bentley.com/docs/table) links to the stories which are already correct as I shared above.

Reply:
This isn't shown in the DropDownMenu docs or stories